egs/iban/s5/local/prepare_lm.sh

```diff
@@ -10,7 +10,7 @@ set -e -o pipefail
 
 local/train_lms_srilm.sh --train-text data/train/text data/ data/srilm
 
-nl -nrz -w10  iban/LM/iban-bp-2012.txt | sort -R > data/local/external_text
+nl -nrz -w10  corpus/LM/iban-bp-2012.txt | sort -R > data/local/external_text
```

Comment:
Yes, that's true but I don't want to have the script to assume the location
-- for example at the CLSP, the location of the data would be somewhere in
/exports/corpora (as it is with tedlium or whatever).
But I see the whole script wasn't built with assumption of that, so I will
fix it later... Let's just leave it now as it is.
y.

On Mon, May 2, 2016 at 4:28 PM, Vijayaditya Peddinti <
notifications@github.com> wrote:

> In egs/iban/s5/local/prepare_lm.sh
> <https://github.com/kaldi-asr/kaldi/pull/744#discussion_r61798290>:
>
> > @@ -10,7 +10,7 @@ set -e -o pipefail
> >
> >  local/train_lms_srilm.sh --train-text data/train/text data/ data/srilm
> >
> > -nl -nrz -w10  iban/LM/iban-bp-2012.txt | sort -R > data/local/external_text
> > +nl -nrz -w10  corpus/LM/iban-bp-2012.txt | sort -R > data/local/external_text
>
> the database is being downloaded to corpus directory. Using a variable for
> $corpus is not necessary.
> … <#m_-8372270746948540648_>
> On Mon, May 2, 2016 at 4:26 PM, jtrmal ***@***.***> wrote: In
> egs/iban/s5/local/prepare_lm.sh <#744 (comment)
> <https://github.com/kaldi-asr/kaldi/pull/744#discussion_r61797923>>: > @@
> -10,7 +10,7 @@ set -e -o pipefail > > local/train_lms_srilm.sh --train-text
> data/train/text data/ data/srilm > > -nl -nrz -w10 iban/LM/iban-bp-2012.txt
> | sort -R > data/local/external_text > +nl -nrz -w10
> corpus/LM/iban-bp-2012.txt | sort -R > data/local/external_text This should
> probably be $corpus or something like that — You are receiving this because
> you authored the thread. Reply to this email directly or view it on GitHub <
> https://github.com/kaldi-asr/kaldi/pull/744/files/9a2c52ded3b2ed10d528d997e60727d67b4511db#r61797923>
>
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/744/files/9a2c52ded3b2ed10d528d997e60727d67b4511db#r61798290>
>